### PR TITLE
Tcp

### PR DESCRIPTION
**Pull request by advitiya7**
### Description

added explanation for four way handshake

### CODE REVIEW

**Comment by advitiya7** [comment]
@wbamberg added information about four way tcp handshake please check
thanks

File: files/en-us/glossary/tcp_handshake/index.md [hunk 1]
```diff
@@ -11,6 +11,18 @@ The three messages transmitted by TCP to negotiate and start a TCP session are n
...
+1. Host A → Host B: FIN flag set.
+2. Host B → Host A: ACK flag set.
+3. Host B → Host A: FIN flag set.
+4. Host A → Host B: ACK flag set.
+These 4 steps are known as a TCP 4-way handshake, which is necessary to terminate a TCP connection. For the termination of the established TCP connection, the following steps are necessary for the process.
+Firstly, from one side of the connection, either from the client or the server the FIN flag will be sent as the request for the termination of the connection.
+ In the second step, whoever receives the FIN flag will then be sending an ACK flag as the acknowledgment for the closing request to the other side.
+ And, at the Later step, the server will also send a FIN flag as the closing signal to the other side.
+In the final step, the TCP, who received the final FIN flag, will be sending an ACK flag as the final Acknowledgement for the suggested connection closing.
+Since the Four main steps are required to close an active connection, so, it is called a four-way handshake.
```
Comment (wbamberg) [inline files/en-us/glossary/tcp_handshake/index.md:25]: This is 100% copied from https://www.geeksforgeeks.org/why-tcp-connect-termination-need-4-way-handshake/. Please, don't do this.

**Comment by wbamberg** [comment]
Please don't copy content from other sources into MDN.

**Comment by wbamberg** [comment]
I appreciate your desire to help out by contributing to MDN, and I'm sorry for being a bit terse above. But it is important from a licensing point of view that contributions are original content, so please keep that in mind. There is a little more about this here: https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Attrib_copyright_license#using_content_from_elsewhere_on_mdn_web_docs.

So it's always better to use your own words when documenting something on MDN.

**Comment by advitiya7** [comment]
I am really sorry for this i didn't know this and in  future it will not happen again I'm really sorry

**Pull request closed without merging**